TestProd/src/com/recapcha/servlet/VerifyRecaptcha.java

```diff
@@ -0,0 +1,84 @@
+package com.recapcha.servlet;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+
+import org.codehaus.jackson.map.ObjectMapper;
+
+
+
+
+public class VerifyRecaptcha {
+	
+	public VerifyRecaptcha() {
+	System.out.println("Verifing...");
+	
+	}
+	
+	public static final String url = "https://www.google.com/recaptcha/api/siteverify";
```

Comment:
You can keep this for now but better way would be to keep this as a property in a property file so that we can change it without touching the code in case google updates this URL.